src/cascadia/TerminalControl/TermControl.h

```diff
@@ -32,6 +32,12 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
     struct TermControl : TermControlT<TermControl>
     {
+        using TimeStamp = uint64_t;
+
+        // represents 0.5 seconds (or 5000 ms). Denoted in microseconds

```

Comment:
```suggestion
        // represents 0.5 seconds (or 500 ms). Denoted in microseconds
```

Half a second is 500ms. 5 seconds is 5000ms.